features/FEATURE_BLE/targets/TARGET_Cypress/TARGET_CYW43XXX/CyVscCmd.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2019 Cypress Semiconductor Corporation
+ * 
```

Comment:
could you add SPDX identifiers to new files?

Reply:
Done.